bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/model/CustomResponseTransformer.java

```diff
@@ -82,24 +82,9 @@ private void updateChargerStartStop(Map<Channel, State> result, String value, Js
         Channel channel = channelProvider.getChannel(CHANNEL_GROUP_CHARGER_COMMANDS, CHANNEL_CHARGER_START_STOP);
         if (channel != null) {
             int val = Integer.parseInt(value);
```

Comment:
```suggestion
            int state = Integer.parseInt(value);
```
If i understand this code, the val is representing some state, so i think state woudl be a better name. You might also refactor it to an enum, but that is maybe out of scope of this PR.